katas/content/superposition_kata/bell_state/solution.md

```diff
@@ -0,0 +1,76 @@
+The first thing we notice is that, unlike in the previous task, we cannot represent this state as a tensor product of two individual qubit states - this goal state is NOT separable. 
+
+> How can we see this? Let's assume that this state can be represented as a tensor product of two qubit states: 
+>
+> $$|\psi_1\rangle \otimes |\psi_2\rangle = (\alpha_1|0\rangle + \beta_1|1\rangle) \otimes (\alpha_2|0\rangle + \beta_2|1\rangle) = \alpha_1\alpha_2|00\rangle + \alpha_1\beta_2|01\rangle + \beta_1\alpha_2|10\rangle + \beta_1\beta_2|11\rangle$$ 
+>
+>In order for this state to be equal to $\frac{1}{\sqrt2}\big(|00\rangle + |11\rangle\big)$, we need to have $\alpha_1\alpha_2 = \beta_1\beta_2 = \frac{1}{\sqrt2}$ and at the same time $\alpha_1\beta_2 = \beta_1\alpha_2 = 0$, which is impossible.
+>
+>This is the first time we encounter the phenomena called **entanglement**, in which the states of the qubits are linked together and can not be considered individually.  
```

Comment:
This sentence might've been written before the Multi-Qubit Systems tutorial was created... Let's rephrase it a bit, since in our learning path this kata comes after that one
```suggestion
>This is the phenomena called **entanglement**, in which the states of the qubits are linked together and can not be considered individually.  
```